db/migrate/20220809130316_add_author_id_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthorIdRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] Again, the reference name should be `author`. Please refer to the above similar comment for the other migration.